packages/devextreme-react/src/core/__tests__/props-updating.test.tsx

```diff
@@ -33,6 +33,7 @@ class NestedComponent extends ConfigurationComponent<{
   a?: number;
   b?: string;
   c?: string;
+  ArrayValue?: Array<unknown> | null;
```

Comment:
Property and field name should starts from a lowercase character

```suggestion
  arrayValue?: Array<unknown> | null;
```